app/models/comment.rb

```diff
@@ -1,6 +1,10 @@
 class Comment < ApplicationRecord
   belongs_to :post
-  belongs_to :user
+  belongs_to :author, class_name:User
```

Comment:
- [ ] [OPTIONAL] Kindly make sure your project has MIT license attached to it. In your Readme, the MIT license link is not working.